mmcv/ops/feature_refine.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch.autograd import Function
+from torch.autograd.function import once_differentiable
+
+from ..utils import ext_loader
+
+ext_module = ext_loader.load_ext(
+    '_ext', ['feature_refine_forward', 'feature_refine_backward'])
+
+
+class FeatureRefineFunction(Function):
```

Comment:
Should we rename it to RefineFeatureFunction? Please @ZwwWayne have a look.

Reply:
How about naming it as RotatedFeatureAlign? RefineFeature may be too general and hard to guess what it really do.

Reply:
Yet. If we rename it as RotatedFeatureAlign, we also rename the file name as well.

Reply:
Done.